### ISSUE TITLE
Event not fired when AndroidLaunchMode = singleTask or singleInstance

### ISSUE THREAD

**Issue by clemorphy** [nordnet/cordova-universal-links-plugin#85]
When compiling with `AndroidLaunchMode`set to `singleTask`or `singleInstance`, if the app has not been launched, it is working. It launches the app and the event is fired.
But if the app has already been launched, it brings the app to foreground, but no event is fired in JS.

Am I missing something ?

I am testing on a Nexus 5 with Android 6.

**Comment by nikDemyankov** [comment]
Sorry, a bit swamped with work right now. Will try to reproduce that tomorrow. For now my guess is that native side is not receiving launch intent when app is resumed. Which would be strange...

**Comment by clemorphy** [comment]
Some news.
I have tested again today, rebuilding the app with `AndroidLaunchMode` set to `singleTask`, and guess what, now it works everytime! Even when the app is already launched, it brings it to foreground, and the JS event is fired.

I don't know what happened the previous times I tested it. I remember that I had rebuilt the app many times, and it was always failing, with `singleTask` or `singleInstance`... So I went back to `standard` mode.

But today I try `singleTask` again, and it works. I didn't change any other parameter.

Maybe that was a temporary problem on my Nexus 5.
